File: tests/data-clumps/test-cases/field-field/simple/positive/typescript/source/Patient.ts
export class Patient {
  public recordId: number;
  public isActive: boolean;
  public visitCount: number;

  constructor(recordId: number, isActive: boolean, visitCount: number) {
    this.recordId = recordId;
    this.isActive = isActive;
    this.visitCount = visitCount;
  }
}

File: tests/data-clumps/test-cases/field-field/simple/positive/java/source/Patient.java
public class Patient {
    public int recordId;
    public boolean isActive;
    public int visitCount;

    public Patient(int recordId, boolean isActive, int visitCount) {
        this.recordId = recordId;
        this.isActive = isActive;
        this.visitCount = visitCount;
    }
}
